Update AboutAccountLevelAPIs.md (#814)
wix-incubator/wix-rest-docs: account-level-apis/guides/AboutAccountLevelAPIs.md

Account owners can use roles to limit team members’ access to various tasks and/or interfaces of the account (for example, blocking access to specific sites or blocking access to adding features that require payment). See [About Team Management](https://support.wix.com/en/article/about-team-management) and [Wix Partners: Default Team Management Roles & Permissions](https://support.wix.com/en/article/default-team-management-roles-permissions#roles-and-permissions-summary-table) for more information.

## Authorization strategies for account level APIs
The APIs in this category require using an [authorization strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies) that utilizes an [API key](https://dev.wix.com/api/rest/getting-started/api-keys):
- Headless projects and apps should use the [Headless admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#headless-admin-with-api-key).

> - Channel partners should use the [Channel admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#channel-admin-with-api-key).

Author: ```suggestion
- Channel partners should use the [Channel admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#channel-admin-with-api-key).  
To learn how to implement this strategy, see the ApiKeyStrategy in [About the Wix Client](https://dev.wix.com/docs/sdk/articles/work-with-the-sdk/about-the-wix-client).  
```

> - Enterprise and Channel partners should use the [Enterprise and Channel admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#enterprise-and-channel-admin-with-api-key).

Author: ```suggestion
- Channel partners should use the [Channel admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#channel-admin-with-api-key).
```

### About API keys
API keys are authentication tools that account owners and co-owners can create, enabling them and any developers they share the key with to make API calls at the account and site levels. 

> **Note:**
> Currently only account owners and co-owners can create API keys. 

> [API keys](https://support.wix.com/en/article/about-wix-api-keys) are created and managed in the [API Keys Manager](https://manage.wix.com/account/api-keys) where you can assign a set of permissions that determine the types of APIs the key can access.

Author: ```suggestion
If you are an account owner or co-owner, you can create and manage [API keys](https://support.wix.com/en/article/about-wix-api-keys) in the [API Keys Manager](https://manage.wix.com/account/api-keys), where you can assign a set of permissions that determine the types of APIs the key can access.
```

![API Keys Manager](../media/APIKeysManager.jpg)

## Terminology:

> - **Account**: A collection of sites and other assets (domains, custom templates, etc.) with associated users.

Author: ```suggestion
- **Account**: Collection of sites and other assets (domains, custom templates, etc.) with associated users.
```

- **API Key**: Authorization key generated in the [API Keys Manager](https://manage.wix.com/account/api-keys).

> - **Reseller**: A strategic partners that offer Wix services to their customers.

Author: ```suggestion
- **Reseller**: A strategic partner that offer Wix services to their customers.
```

Account owners can use roles to limit team members’ access to various tasks and/or interfaces of the account (for example, blocking access to specific sites or blocking access to adding features that require payment). See [About Team Management](https://support.wix.com/en/article/about-team-management) and [Wix Partners: Default Team Management Roles & Permissions](https://support.wix.com/en/article/default-team-management-roles-permissions#roles-and-permissions-summary-table) for more information.

## Authorization strategies for account level APIs
The APIs in this category require using an [authorization strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies) that utilizes an [API key](https://dev.wix.com/api/rest/getting-started/api-keys):

> - Headless projects and apps should use the [Headless admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#headless-admin-with-api-key).

Author: ```suggestion
- Headless projects and apps should use the [Headless admin with API key strategy](https://dev.wix.com/docs/sdk/articles/get-started/authorization-strategies#headless-admin-with-api-key).  
To learn how to implement this strategy, see [Create a Client with an API Key](https://dev.wix.com/docs/go-headless/coding/java-script-sdk/admin/create-a-client-with-an-api-key).
```
